doc/install/install-ceph-gateway.rst

```diff
@@ -164,7 +164,7 @@ it is not possible to configure the radosgw to listen on both
 http and https, you must pick only one.  So::
 
  [client.rgw.gateway-node1]
- rgw_frontends = civetweb port=443s ssl_certificate=/etc/ceph/private/keyandcert.pem
+ rgw_frontends = civetweb port=443s ssl_certificate=/etc/ceph/private/keyandcert.pem ssl_protocol_version=4 ssl_cipher_list=EECDH+AESGCM:EDH+AESGCM:AES256+EECDH:AES256+EDH:!LOW:!RC4:!MD5:!3DES
```

Comment:
Shouldn't the sample config have a minimal config?  In general the user shouldn't be manually specifying the ciphers, right?

Reply:
Right, the user doesn't need to specify these because those are the default with this patch. But I wanted to include some example to show how to change the ciphers.